### PR TITLE
add error message components

### PR DESCRIPTION
**Pull request by doramatadora**
Adds:
- client-side class: `ConsentMessage`
- scss mixins: `successMessage` and `errorMessage`
- handlebars templates
- `/message` [route](http://localhost:5005/message) in demos
- bower_components resolution in demos webpack config

**Demo**
screen shot 2018-04-23 at 14 51 46

### CODE REVIEW

File: src/js/client/message.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		this.element.style.display = 'none';
+	}
+
+	public show() {
+		this.element.style.display = 'block';
+	}
+
+	public init(show?: boolean) {
+		this._message.init();
+		if(show) {
```
Comment (jonnywyatt) [inline src/js/client/message.ts]: Do you need both 'show' param and 'hideOnInit' option, a single boolean should suffice?
I'm confused as to the default behaviour, does it show or hide by default?

Comment (doramatadora) [reply src/js/client/message.ts]: good point, I'll update so it's smarter

File: templates/changes-saved-message.html [hunk 2]
```diff
@@ -1,9 +1,11 @@
-<div class="margin-top-x4">
-	<div class="consent-msg consent-msg--alert-inner consent-msg--success">
-		<div class="consent-msg__container">
-			<div class="consent-msg__content">
-				<p class="consent-msg__content-main">
-					<span class="consent-msg__content-highlight">Your changes have been saved.</span>
+<div class="margin-top-x4 consent-msg">
+	<div class="o-message o-message--alert-inner o-message--success" data-o-component="o-message">
```
Comment (jonnywyatt) [inline templates/changes-saved-message.html]: It looked like silent mode wasn't turned off for o-message? Better to keep it on and bring in just the CSS we need with mixins

Comment (doramatadora) [reply templates/changes-saved-message.html]: I thought it was silent by default: https://github.com/Financial-Times/o-message#sass

File: src/scss/components/alert-message.scss [hunk 3]
```diff
@@ -0,0 +1,18 @@
+@import 'o-message/main';
+
+@mixin alertMessage($type, $status) {
+	@include oMessage($class: 'o-message', $types: $type, $status: $status);
+	// TODO: remove this override
+	// once Origami have addressed https://github.com/Financial-Times/o-message/issues/26
```
Comment (pixelandpage) [inline src/scss/components/alert-message.scss]: Do we have a hygiene ticket for this so we don't forget about it?

**Pull request merged**